src/main/features/response/form/models/statement-of-means/bankAccounts.ts

```diff
@@ -0,0 +1,28 @@
+import { MultiRowForm } from 'forms/models/multiRowForm'
+import { BankAccountRow } from 'response/form/models/statement-of-means/bankAccountRow'
+
+export const MAX_NUMBER_OF_ROWS: number = 10
+export const INIT_ROW_COUNT: number = 2
+
+export class BankAccounts extends MultiRowForm<BankAccountRow> {
+
+  static fromObject (value?: any): BankAccounts {
+    if (!value) {
+      return value
+    }
+
+    return new BankAccounts(value.rows ? value.rows.map(BankAccountRow.fromObject) : [])
+  }
+
+  createEmptyRow (): BankAccountRow {
```

Comment:
do we need this? Can't it come from the super class?

Reply:
If you find the way to create an object when you have `T` not constructor.

Reply:
How will an abstraction know how to create a concrete instance? Unless you meant to return `{ }` and 🤞 it will work? It could work.

Reply:
abstract class is a genric, so it has `T`. Unfortunately I haven't found anything that would work here :/ 

@timja-kainos I'm happy to refactor it, but please show me how :)